pages/_app.js

```diff
@@ -1,13 +1,12 @@
 import "../styles/globals.css";
 import Meta from "../components/Meta";
-import Middle from "../components/Middle";
 
 function MyApp({ Component, pageProps }) {
   return (
     <>
       <Meta />
       <Component {...pageProps} />
-      <Middle />
```

Comment:
Moved to pages/index.js as this component is specific to the first page

Reply:
looks good